src/editors/containers/ProblemEditor/data/mockData/editorTestData.js

```diff
@@ -0,0 +1,62 @@
+export const checkboxesWithFeedbackAndHints = {
+  solution: `<div class="detailed-solution">
+    <p>Explanation</p>
+    <p>
+        You can form a voltage divider that evenly divides the input
+        voltage with two identically valued resistors, with the sampled
+        voltage taken in between the two.
+    </p>
+    <p><img src="/static/images/voltage_divider.png" alt=""></img></p>
+  </div>`,
+  question: '<p>You can use this template as a guide to the simple editor markdown and OLX markup to use for checkboxes with hints and feedback problems. Edit this component to replace this template with your own assessment.</p><label>Add the question text, or prompt, here. This text is required.</label><em>You can add an optional tip or note related to the prompt like this.</em>',
+  answers: {
+    A: 'a correct answer',
+    B: 'an incorrect answer',
+    C: 'an incorrect answer',
+    D: 'a correct answer',
+  },
+};
+
+// export const multipleChoiceWithoutAnswers = {
+//   solution: '',
+//   question: '',
+//   answers: { A: '' },
+// };
+
+export const dropdownWithFeedbackAndHints = {
+  solution: '',
```

Comment:
why do only come of these have solutions?

Reply:
I based on the data that was already present in `olxTestData`